samples/application-conf/README.md

```diff
@@ -1,12 +1,12 @@
 # Application Configuration
 This folder contains application specific configuration properties used by the zAppBuild Groovy build and utility scripts. It is intended to be copied as a high level folder in the application repository or main application repository if the application source files are distributed across multiple repositories. Once copied to the application repository, users should review the default property files and modify any values as needed.
 
-At the beginning of the build, the `application-conf/application.properties` file will automatically be loaded into the [DBB BuildProperties class](https://www.ibm.com/support/knowledgecenter/SS6T76_1.0.4/scriptorg.html#build-properties-class). Use the `applicationPropFiles` property (see table below) to load additional application property files.
+At the beginning of the build, the `application-conf/application.properties` file will automatically be loaded based on the definition of `applicationConfDir` [property](../../build-conf/build.properties) into the [DBB BuildProperties class](https://www.ibm.com/docs/en/dbb/2.0?topic=apis-build-properties#build-properties-class). Use the `applicationPropFiles` property (see table below) to load additional application property files.
```

Comment:
Maybe we can say that it's "by default, zAppBuild is configured to look at `application-conf/application.properties` but this can be changed. Something like:

At the beginning of the build, the `application.properties` file will automatically be searched and loaded if it exists into the [DBB BuildProperties class](https://www.ibm.com/docs/en/dbb/2.0?topic=apis-build-properties#build-properties-class). The `application.properties` file is by default searched in the `application-conf` folder of the application, but this can be modified through the `applicationConfDir` [property](../../build-conf/build.properties).